src/frontend/src/components/common/Pagination/index.tsx

```diff
@@ -0,0 +1,141 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faChevronDown } from '@fortawesome/free-solid-svg-icons';
+import { faAngleLeft } from '@fortawesome/free-solid-svg-icons';
+import { faAngleRight } from '@fortawesome/free-solid-svg-icons';
+
+type PaginationProps = {
+  currentPage: number;
+  totalEntries: number;
+  pageSize: number;
+  changePage: (page: number) => void;
+  changePageSize: (pageSize: number) => void;
+};
+
+const Pagination = ({
+  currentPage,
+  totalEntries,
+  pageSize,
+  changePage,
+  changePageSize,
+}: PaginationProps) => {
+  const PageSizeOptions = [10, 20, 30, 50, 100];
+  return (
+    <div className="pagination">
+      <div className="pagination__container">
+        <div className="pagination__container__left">
+          <div className="pagination__container__left__text">
+            <span>
+              Showing {(currentPage - 1) * pageSize + 1} to{' '}
+              {totalEntries < pageSize * currentPage
+                ? totalEntries
+                : currentPage * pageSize}{' '}
```

Comment:
remove empty string at the end

Reply:
the empty string is removed at the end after the word 'entries'. 
Showing 1 to 10 of 11 entries